lib/output.js

```diff
@@ -410,6 +410,12 @@ function OutputStream(options) {
     AST_Node.DEFMETHOD("print_to_string", function(options){
         var s = OutputStream(options);
         this.print(s);
+        if (options === undefined) {
```

Comment:
If `options` is not set then that means use default output options where `comments` is false. I think it's fine. Fewer conditions in this code path.